mrtt-ui/src/components/SiteInterventions.js

```diff
@@ -0,0 +1,162 @@
+import { useState } from 'react'
+import axios from 'axios'
+import { useParams } from 'react-router-dom'
+import {
+  useForm,
+  // useFieldArray,
```

Comment:
guessing you will use this commented out code in the next step or so? If not, can you remove it and others?